website/src/components/CodeSnippet.jsx

```diff
@@ -1,20 +1,27 @@
 /* eslint-disable react/jsx-key */
 
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import Highlight, { defaultProps } from 'prism-react-renderer';
 
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import useThemeContext from '@theme/hooks/useThemeContext';
 
 function CodeSnippet({ code }) {
+  const [mounted, setMounted] = useState(false);
+
+  useEffect(() => {
+    setMounted(true);
+  }, []);
```

Comment:
could you explain the rationale behind this? I understand what it does (remounts the `<Highlight/>`) but I'm wondering why this is needed here.

Reply:
It just ensures that the component and its contents are mounted _after_ the code style is loaded.

Reply:
what the code style (css for that?) is and how it relates to mounting?